setup.py

```diff
@@ -4,17 +4,16 @@
 import subprocess
 
 from setuptools import setup, find_packages
-import setuptools.command.develop
-import setuptools.command.install
 
 cwd = os.path.dirname(os.path.abspath(__file__))
 
-version = '0.0.7'
+version = '0.0.8'
 try:
-    from datetime import date
-    today = date.today()
-    day = today.strftime("b%d%m%Y")
-    version += day
+    if not os.getenv('RELEASE'):
+        from datetime import date
+        today = date.today()
+        day = today.strftime("b%Y%m%d")
+        version += day
```

Comment:
Should we add an if/else flag to disable this for official/stable versions?  

The version should not be `version += day` for those versions, currently I think one has to manually remove this line of code to build official versions.